### ISSUE TITLE
PageHead border needs a way of turning it off

### ISSUE THREAD

**Issue by falsefalse** [toptal/picasso#2955]
**Component**
[PageHead](https://picasso.toptal.net/?path=/story/components-pagehead--pagehead)

**Is your feature request related to a problem? Please describe.**
For certain pages/conditions we want to be able to have a page header without bottom border — so it does not clash visually with the next blocks border.

**Describe the solution you'd like**
A `noBorder = false` prop that can be set to `true` when border is not desired. Implementation-wise, it would control if `root::after` style is applied.

**Describe alternatives you've considered**
Can't see any.

**Additional context**
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/38544/176660617-8f52b0e4-acc5-48f4-a3ef-0d292dca33fb.png">

**Comment by LashaJini** [comment]
For me, `noBorder` prop would mean to enable/disable all the 4 borders from the component. We can work on the name. @falsefalse Please ask in the base channel, if base team is fine with it then so do we.

**Comment by falsefalse** [comment]
> For me, `noBorder` prop would mean to enable/disable all the 4 borders from the component.

But it does not have four borders it has one. Also, `noBorders` ≠ `noBorder`.

Anyhow, I don't mind any name that makes everyone happy. I just need a way to turn off the hardcoded border.

**Comment by falsefalse** [comment]
Slack [discussion](https://toptal-core.slack.com/archives/CCC3GP6CC/p1656587895873919)

> [Milos Milikic]:  I’m not sure if we used page head without the border anywhere. We either use it with it or do not use it at all. But for me, it makes sense to have it customizable.

**Comment by LashaJini** [comment]
Could you please create PR for it :bow:

**Comment by falsefalse** [comment]
@LashaJini nope, sorry, not sooner than couple of weeks from now, we've got rather intense sprint going on

**Comment by falsefalse** [comment]
Still no time for that 😬  Any chance this gets fixed @LashaJini?
